taskManager/src/main/java/com/kunmi/taskManager/taskManager/services/ProjectServiceImpl.java

```diff
@@ -0,0 +1,111 @@
+package com.kunmi.taskManager.taskManager.services;
+
+import com.kunmi.taskManager.taskManager.records.*;
+import com.kunmi.taskManager.taskManager.exceptions.ProjectNotFoundException;
+import com.kunmi.taskManager.taskManager.exceptions.UserNotFoundException;
+import com.kunmi.taskManager.taskManager.models.Project;
+import com.kunmi.taskManager.taskManager.models.User;
+import com.kunmi.taskManager.taskManager.repositories.ProjectRepository;
+import com.kunmi.taskManager.taskManager.repositories.UserRepository;
+import jakarta.transaction.Transactional;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.UUID;
+
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class ProjectServiceImpl implements ProjectService {
+
+    private final UserService userService;
+    private final UserRepository userRepository;
+    private final ProjectRepository projectRepository;
+
+    @Override
+    public ProjectListResponseRecord findAll(String email) {
+        User user = getCurrentUser(email);
+
+        log.info("Fetching projects for user: {}", user.getEmail());
+        List<Project> projects = projectRepository.findByUserId(user.getId());
+        log.info("Found {} projects for user {}", projects.size(), user.getEmail());
+
+        if (projects.isEmpty()) {
+            log.info("No projects found for user: {}", user.getEmail());
+            return new ProjectListResponseRecord("No projects found for user: " + user.getEmail());
+        }
+
+        List<ProjectRecord> projectRecords = projects.stream()
+                .map(project -> new ProjectRecord(project.getId().toString(), project.getName())
+                ).toList();
+
+        return new ProjectListResponseRecord(projectRecords, null);
+    }
+
+    @Override
+    public GetProjectResponseRecord find(String email, UUID id) {
+        User currentUser = getCurrentUser(email);
+        Project project = projectRepository.findById(id).orElseThrow(() ->
+                new ProjectNotFoundException("Project not found for user: " + currentUser.getEmail()));
+
+        if (!project.getUser().getId().equals(currentUser.getId())) {
+            throw new ProjectNotFoundException("Project not found for user:" + currentUser.getEmail());
+        }
+
+        return new GetProjectResponseRecord(project.getId(), project.getName());
+    }
+
+    @Override
+    @Transactional
+    public ProjectResponseRecord create(ProjectRequestRecord projectRequestRecord, String email) {
```

Comment:
All this messages in response are too much. Just return new record if success and return error code(maybe with description) if error